### PR TITLE
encoded ISic030008-ISic030013

### PR DESCRIPTION
**Pull request by savannahneibart**

### CODE REVIEW

**Review by valentinamignosa — CHANGES_REQUESTED** [review]
ISic030008)
1. you can delete my comments in the files if you have entered the requested information;
2. This section indicates the generic place of origin of the document, so:`<placeName type="ancient" ref="http://pleiades.stoa.org/places/462489">Selinus</placeName>		
                                <placeName type="modern" ref="http://sws.geonames.org/2523164">Selinunte</placeName>
                                <geo>37.58337, 12.82522</geo>`. The specific coordinates of the place of discovery should be indicated in`<provenance>` where you can add: `<provenance type="found" subtype="discovered">Gaggera, Sanctuary of Malophoros<geo>37.586763, 12.816874</geo>.</provenance>`;
3. origdate: we usually indicate the dates as 499—450 BCE (and in brackets if the dating is from another editor you can indicate something like 'so Bettarini on letters' etc.)
4. text: resp="#SN"
5. text: the column number is usually indicated by a, b, c etc. (not i, ii, ...)
6. text, line 2: on epsilon and omicron there is a macron (not a supraline): ε̄́ ο̄ (if you have troubles with these combined characters let me know)
7. text: I would add a vacat to lines 3 and 4 of column b `<space extent="unknown" unit="character"/>`
8. text: about the diple, your interpretation is correct, but note: `<abbr>
<am><g type="diple"/></am>
</abbr>
          <ex cert="low">γράφο̄</ex></expan>`
9. commentary: you can open <p> at line 37 and close </p> at line 39
10. bibliography: insert a comma followed by a space between the page number and the number of the inscription and space after 'no.' (197-199, no. 27)

**Review by valentinamignosa — CHANGES_REQUESTED** [review]
as previous, you can delete my comments

File: inscriptions/ISic030010.xml [hunk 1]
```diff
@@ -109,7 +115,7 @@
                                 <placeName type="modern">Selinunte</placeName>
                                 <geo>37.5853775, 12.8298666</geo>
                             </origPlace>
-                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0600" notAfter-custom="-0400" precision="low">6th—5th century BCE</origDate> <!-- check date -->
+                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0500" notAfter-custom="-0450">First half V cent. BCE</origDate> <!-- check date -->
```
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:118]: as previous, 500—450 BCE

File: inscriptions/ISic030010.xml [hunk 2]
```diff
@@ -109,7 +115,7 @@
                                 <placeName type="modern">Selinunte</placeName>
```
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:115]: `<placeName type="ancient" ref="http://pleiades.stoa.org/places/462489">Selinus</placeName>		
                                <placeName type="modern" ref="http://sws.geonames.org/2523164">Archaeological site of Selinunte</placeName>
                                <geo>37.58337, 12.82522</geo>`

File: inscriptions/ISic030010.xml [hunk 3]
```diff
@@ -109,7 +115,7 @@
...
                                 <geo>37.5853775, 12.8298666</geo>
                             </origPlace>
-                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0600" notAfter-custom="-0400" precision="low">6th—5th century BCE</origDate> <!-- check date -->
+                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0500" notAfter-custom="-0450">First half V cent. BCE</origDate> <!-- check date -->
                         </origin>
                         <provenance type="found">Found in the archaeological area of Selinunte<geo>37.5853775, 12.8298666</geo>.</provenance> <!-- check provenance -->
```
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:120]: It was found in the necropolis in Contrada Buffa. Please leave the generic coordinates for Selinunte for now and add a comment to flag it out

File: inscriptions/ISic030010.xml [hunk 4]
```diff
@@ -177,21 +184,23 @@
...
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TS"> <!-- add text, apparatus, translation and commentary -->
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#SN"> <!-- add text, apparatus, translation and commentary -->
                 <ab>
-                    <lb n="1"/>
-                    <lb n="2"/>
+                    <lb n="1"/> Χάμις ἐπέ<unclear>χ</unclear><hi rend="supraline">ο</hi>
```
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:193]: delete space before Χάμις; add a macron on omicron ο̄(delete supraline)

File: inscriptions/ISic030010.xml [hunk 5]
```diff
@@ -177,21 +184,23 @@
...
+                    <lb n="2"/>κα<supplied reason="omitted">ὶ</supplied> νόσ<hi rend="supraline">ο</hi> ῥοπᾶι
```
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:194]: add a macron on omicron ο̄(delete supraline)

File: inscriptions/ISic030010.xml [hunk 6]
```diff
@@ -177,21 +184,23 @@
...
+                    <lb n="3"/>ἐπ’ ἀτελείαι ℎο
+                    <lb n="4" break="no"/><unclear>τ</unclear><supplied reason="lost">ι</supplied> κα<gap reason="lost" quantity="2" unit="character"/><unclear>ε</unclear><gap reason="lost" quantity="2" unit="character"/><unclear>ο</unclear>ντι.
```
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:196]: `<orig>ΚΑ</orig><gap reason="lost" quantity="2" unit="character"/><orig><unclear>Ε</unclear></orig><gap reason="lost" quantity="2" unit="character"/><orig><unclear>Ο</unclear>ΝΤΙ</orig>.`

File: inscriptions/ISic030010.xml [hunk 7]
```diff
@@ -201,14 +210,17 @@
                     </app>
                 </listApp>
             </div>
-            <div type="translation" xml:lang="en">
-                <p></p>
+            <div type="translation" xml:lang="fr">
+                <p>Je te maintiens dans la crise fatale de la maladie afin que tu ne puisses atteindre tes objectifs.(Dubois)</p>
             </div>
             <div type="translation" xml:lang="it">
                 <p></p>
```
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:217]: 'Chamis ti conduco alla fase critica della malattia perché tu non possa portare a termine ...'

File: inscriptions/ISic030010.xml [hunk 8]
```diff
@@ -201,14 +210,17 @@
...
             </div>
             <div type="commentary" xml:lang="en">
-                <p></p>
+                <p>Il doit s'agir d'une imprécation contre un grabataire dont l'impotence réjouit le rédacteur de la tablette. (Dubois)</p>
```
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:220]: as previous, you can use <p> here and close it at the end of the commentary

File: inscriptions/ISic030010.xml [hunk 9]
```diff
@@ -218,10 +230,50 @@
                         <citedRange>66</citedRange>
                         <ptr target="http://zotero.org/groups/382445/items/VC9IHZAR"/>
                     </bibl>
+                    <bibl>
+                        <author>Brugnone</author>
+                        <date>1976</date>
+                        <citedRange>81-83 no.4</citedRange>
```
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:236]: as previous, '81-83, no. 4' (also edit other entries)

File: inscriptions/ISic030011.xml [hunk 10]
```diff
@@ -109,9 +113,10 @@
                                 <placeName type="modern">Selinunte</placeName>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:113]: as previous, add modern and ancient place (Selinunte/Selinus) with generic coordinates

File: inscriptions/ISic030011.xml [hunk 11]
```diff
@@ -109,9 +113,10 @@
...
                                 <geo>37.5853775, 12.8298666</geo>
                             </origPlace>
-                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0600" notAfter-custom="-0400" precision="low">6th—5th century BCE</origDate> <!-- check date -->
+                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" cert="low" notBefore-custom="-0475" notAfter-custom="-0450">475—450 BCE</origDate>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:116]: you can add the information that the dates are from Jordan in plain text: 475—450 BCE (so Jordan on letters)

File: inscriptions/ISic030011.xml [hunk 12]
```diff
@@ -109,9 +113,10 @@
                                 <placeName type="modern">Selinunte</placeName>
                                 <geo>37.5853775, 12.8298666</geo>
                             </origPlace>
-                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0600" notAfter-custom="-0400" precision="low">6th—5th century BCE</origDate> <!-- check date -->
+                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" cert="low" notBefore-custom="-0475" notAfter-custom="-0450">475—450 BCE</origDate> 
+                            <!-- dates from Jordan-->
                         </origin>
-                        <provenance type="found">Found in the archaeological area of Selinunte<geo>37.5853775, 12.8298666</geo>.</provenance> <!-- check provenance -->
+                        <provenance type="found">Found in the archaeological area of Selinunte, in the Malophoros Sanctuary<geo>37.585378, 12.829867</geo>.</provenance>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:119]: the correct geo ref is: 37.586763, 12.816874 (it is in pleiades: https://pleiades.stoa.org/places/502577471)

File: inscriptions/ISic030011.xml [hunk 13]
```diff
@@ -161,6 +166,7 @@
                 <change when="2020-10-05" who="#SS">Simona Stoyanova normalised Unicode</change>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:166]: status="draft" (change/check also in the other files)

File: inscriptions/ISic030011.xml [hunk 14]
```diff
@@ -218,10 +231,50 @@
...
                         <ptr target="http://zotero.org/groups/382445/items/VC9IHZAR"/>
                     </bibl>
+                    <bibl>
+                        <author>Ferri</author>
+                        <date>1944-1945</date>
+                        <citedRange>VII.5-6</citedRange>
+                        <ptr target="https://www.zotero.org/groups/382445/isicily/items/T24IFS3N"
+                    </bibl>
+                    <bibl type="bulletin" n="SEG">
+                        <citedRange>16.574</citedRange>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:241]: 16.0574

File: inscriptions/ISic030011.xml [hunk 15]
```diff
@@ -218,10 +231,50 @@
...
+                        <ptr target="http://zotero.org/groups/382445/items/F2GG87EQ"/>
+                    </bibl>
+                    <bibl>
+                        <author>Dubois</author>
+                        <date>1989</date>
+                        <citedRange>43-44 no.33</citedRange>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:247]: as previous, 43-44, no. 33

File: inscriptions/ISic030011.xml [hunk 16]
```diff
@@ -218,10 +231,50 @@
...
+                        <ptr target="https://www.zotero.org/groups/382445/isicily/items/RWXNUFEX"/>
+                    </bibl>
+                    <bibl>
+                    <author>Lopez-Jimeno</author>
+                    <date>1991</date>
+                    <citedRange>no.13</citedRange>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:253]: no. 13 (change/check other entries)

File: inscriptions/ISic030011.xml [hunk 17]
```diff
@@ -177,24 +183,30 @@
...
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TS"> <!-- add text, apparatus, translation and commentary -->
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#SN"> <!-- add text, apparatus, translation and commentary -->
                 <ab>
-                    <lb n="1"/>
-                    <lb n="2"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><unclear>τ</unclear>ο͂ι Σο̄π<unclear>ά</unclear>τ<unclear>ρ</unclear>ο̄ι <space extent="unknown" unit="character"/>
+                    <lb n="2"/>ΓΥϙ<gap reason="lost" quantity="3" unit="character"/><unclear>ο</unclear>ς κ<unclear>α</unclear>ὶ ℎα Σο̄π<unclear>ά</unclear>τρο̄ γλο͂<supplied reason="lost"><choice><orig>σα</orig><reg>σσα</reg></choice></supplied>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:193]: <orig>ΓΥϘ</orig>

Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:193]: `<choice><orig>γλο͂<supplied reason="lost">σα</supplied></orig><reg>γλο͂<supplied reason="lost">σσα</supplied></reg></choice>`

File: inscriptions/ISic030011.xml [hunk 18]
```diff
@@ -177,24 +183,30 @@
...
+                    <lb n="3"/>Φρῦνις Νοαβ<unclear>αρ</unclear>ίλο̄ καὶ ℎα <supplied reason="lost">Φρύνιος <choice><orig>γ</orig></choice></supplied>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:194]: for the lemmatisation which will be done later, it is necessary to add the <supplied> around each word. Same for the <choice><orig>/<reg> (but check if the validation fails with the line break):
`<supplied reason="lost">Φρύνιος</supplied> <choice><orig><supplied reason="lost">γ</supplied>
<lb n="4" break="no"/>λο͂σ<unclear>α</unclear></orig><reg><supplied reason="lost">γ</supplied>λο͂σσα</reg></choice> <surplus><choice><orig>γλο͂σα</orig><reg>γλο͂σσα</reg></choice></surplus>`

File: inscriptions/ISic030011.xml [hunk 19]
```diff
@@ -177,24 +183,30 @@
...
+                    <lb n="4" break="no"/><choice><orig>λο͂σ<unclear>α</unclear></orig><reg>λο͂σσα</reg></choice> <surplus><choice><orig>γλο͂σ<unclear>α</unclear></orig><reg>γλο͂σσα</reg></choice></surplus> ∶ ἀ<unclear>ν</unclear>ε<gap reason="lost" extent="unknown" unit="character" cert="low"/>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:195]: `ἀ<unclear>ν</unclear>ε<gap reason="illegible" extent="unknown" unit="character"/>`

File: inscriptions/ISic030011.xml [hunk 20]
```diff
@@ -177,24 +183,30 @@
...
+                    <lb n="5"/>ΜΕ<unclear>Κ</unclear>ΟΛΥ<unclear>Ε</unclear> συνδικε<gap reason="lost" extent="unknown" unit="character"/>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:196]: when letters cannot be interpreted in any way they are placed inside orig in capitals:`<orig>ΜΕ<unclear>Κ</unclear>ΟΛΥ<unclear>Ε</unclear></orig>`

File: inscriptions/ISic030011.xml [hunk 21]
```diff
@@ -177,24 +183,30 @@
...
+                    <lb n="6"/>τὰς Γ<gap reason="lost" quantity="3" unit="character"/>ΛΟ <space extent="unknown" unit="character"/> Λ<gap reason="lost" extent="unknown" unit="character"/>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:197]: as previous: `<orig>Γ</orig><gap reason="lost" quantity="3" unit="character"/><orig>ΛΟ</orig><space extent="unknown" unit="character"/><orig>Λ</orig>`

File: inscriptions/ISic030011.xml [hunk 22]
```diff
@@ -177,24 +183,30 @@
...
                 <listApp>
-                    <app>
-                        <note>Text after ...</note>
+                    <app loc="4">
+                        <lem>ἀ<unclear>ν</unclear>ε<gap reason="lost" extent="unknown" unit="character"/></lem>
+                        <rdg source="tdb1989">ἀ<unclear>ν</unclear>ε<supplied reason="lost"><choice><orig>στραμένα</orig><reg>στραμμένα</reg></choice></supplied><gap reason="lost" extent="unknown" unit="character"/></rdg>
+                        <note>Alternate reading by Arena.</note>
                     </app>
                     <app>
-                        <note></note>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:197]: add the chosen edition before the actual apparatus

File: inscriptions/ISic030011.xml [hunk 23]
```diff
@@ -177,24 +183,30 @@
...
+                    
                 </ab>
             </div>
             <div type="apparatus">
                 <listApp>
-                    <app>
-                        <note>Text after ...</note>
+                    <app loc="4">
+                        <lem>ἀ<unclear>ν</unclear>ε<gap reason="lost" extent="unknown" unit="character"/></lem>
+                        <rdg source="tdb1989">ἀ<unclear>ν</unclear>ε<supplied reason="lost"><choice><orig>στραμένα</orig><reg>στραμμένα</reg></choice></supplied><gap reason="lost" extent="unknown" unit="character"/></rdg>
```
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:205]: for now we are not tagging the sources in the apparatus. Leave it as it is but for other files just use: 
`<app loc="1">
       <note>Arena: ....</note>
</app>`

File: inscriptions/ISic030011.xml (hunk 14 above)
Comment (valentinamignosa) [inline inscriptions/ISic030011.xml:241]: 16.0574

File: inscriptions/ISic030013.xml [hunk 24]
```diff
@@ -74,30 +76,30 @@
...
-                                <condition/>
+                                <condition ana="#condition.complete"/>
                             </supportDesc>
                             <layoutDesc>
                                 <layout>
-                                    <rs ana="#execution.engraved" ref="">engraved</rs> <!-- add ref to eagle vocabulary -->
-                                    <p></p>
-                                    <damage/>
+                                    <rs ana="#execution.engraved" ref="https://www.eagle-network.eu/voc/writing/lod/3.html">engraved</rs> 
+                                    <p>L'imprecazione è scritta in direzione progressiva, i nomi dei defissi invece sono scritti in direzione regressiva. (Arena)</p>
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:92]: The curse is written left-to-right while the names of the victims are written right-to-left.

File: inscriptions/ISic030013.xml [hunk 25]
```diff
@@ -109,9 +111,9 @@
                                 <placeName type="modern">Selinunte</placeName>
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:111]: as previous, add `<placeName type="ancient" ref="http://pleiades.stoa.org/places/462489">Selinus</placeName>		
                                <placeName type="modern" ref="http://sws.geonames.org/2523164">Selinunte</placeName>
                                <geo>37.58337, 12.82522</geo>`

File: inscriptions/ISic030013.xml [hunk 26]
```diff
@@ -109,9 +111,9 @@
...
                                 <geo>37.5853775, 12.8298666</geo>
                             </origPlace>
-                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0600" notAfter-custom="-0400" precision="low">6th—5th century BCE</origDate> <!-- check date -->
+                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0450" notAfter-custom="-0450">450 BCE</origDate> <!-- from Arena-->
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:114]: dating cannot actually be done ad annum, it is approximate. In such cases, we suggest entering a range according to what the editor says. In this case: 475-450

File: inscriptions/ISic030013.xml [hunk 27]
```diff
@@ -109,9 +111,9 @@
...
                         </origin>
-                        <provenance type="found">Found in the archaeological area of Selinunte<geo>37.5853775, 12.8298666</geo>.</provenance> <!-- check provenance -->
+                        <provenance type="found">Found in the archaeological area of Selinunte, in the Malophoros Sanctuary<geo>37.585378, 12.829867</geo>.</provenance>
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:116]: correct coordinates: 37.586763, 12.816874

File: inscriptions/ISic030013.xml [hunk 28]
```diff
@@ -177,38 +180,77 @@
...
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TS"> <!-- add text, apparatus, translation and commentary -->
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#SN"> <!-- add text, apparatus, translation and commentary -->
                 <ab>
-                    <lb n="1"/>
-                    <lb n="2"/>
+                    <lb n="1"/> ἐ<unclear>π</unclear>ὶ ἐξο̄<unclear>λ</unclear>είαι καὶ αὐτο͂ν καὶ γενεᾶς· Νίκυλλος
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:189]: delete space before epsilon

File: inscriptions/ISic030013.xml [hunk 29]
```diff
@@ -177,38 +180,77 @@
...
             <div type="apparatus">
                 <listApp>
                     <app>
-                        <note>Text after ...</note>
+                        <note>Text after Bettarini</note>
                     </app>
-                    <app>
-                        <note></note>
+                    <app loc="1">
+                        <lem>ἐ<unclear>π</unclear>ὶ</lem>
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:204]: leave as it is for now

File: inscriptions/ISic030013.xml [hunk 30]
```diff
@@ -177,38 +180,77 @@
...
     <text>
         <body>
-            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TS"> <!-- add text, apparatus, translation and commentary -->
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#SN"> <!-- add text, apparatus, translation and commentary -->
                 <ab>
-                    <lb n="1"/>
-                    <lb n="2"/>
+                    <lb n="1"/> ἐ<unclear>π</unclear>ὶ ἐξο̄<unclear>λ</unclear>είαι καὶ αὐτο͂ν καὶ γενεᾶς· Νίκυλλος
+                    <lb n="2"/>Καπόσο̄, Δένδιλος Μάμονος, Αἴνο̄ν Β<unclear>λ</unclear>έπονος,
+                    <lb n="3"/>Ξένιος Ἀπόντιος ℎɛ̄ρακλείδας, Σαῦρις, <seg style="textdirection:r-to-l">ανις
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:191]: H

File: inscriptions/ISic030013.xml [hunk 31]
```diff
@@ -177,38 +180,77 @@
...
+                   <lb n="4" break="no"/>Ἄθ</seg> Ταμμάρο̄ ℎɛ̄ρακλείδας, Ῥας Φαρμαυα, <seg style="textdirection:r-to-l">ν
+                    <lb n="5" break="no"/>Δίο̄</seg> Πιάκιος, Πιθθίας, Χαίο̄ν, <choice><orig>Μαμαρειṑι</orig><reg>Μαμμαρειṑι</reg></choice> <seg style="textdirection:r-to-l">τα
+                    <lb n="6" break="no"/>Ζο<unclear>ζ</unclear></seg>, Ἀγάθυλλος Ξένιος ℎɛ̄ρακλείδας, <seg style="textdirection:r-to-l">τος
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:194]: Hɛ̄ρακλείδας

File: inscriptions/ISic030013.xml [hunk 32]
```diff
@@ -218,10 +260,53 @@
                         <citedRange>69</citedRange>
                         <ptr target="http://zotero.org/groups/382445/items/VC9IHZAR"/>
                     </bibl>
+                    <bibl>
+                        <author>Gabrici</author>
+                        <date>1927</date> 
+                        <citedRange>no.15</citedRange>
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:266]: as previous, no. 15 (change/check other entries)

File: inscriptions/ISic030013.xml [hunk 33]
```diff
@@ -218,10 +260,53 @@
...
                         <ptr target="http://zotero.org/groups/382445/items/VC9IHZAR"/>
                     </bibl>
+                    <bibl>
+                        <author>Gabrici</author>
+                        <date>1927</date> 
+                        <citedRange>no.15</citedRange>           
+                        <ptr target = "https://www.zotero.org/groups/382445/isicily/items/SP4NJIAA"/>
+                    </bibl>
+                    <bibl type="bulletin" n="SEG">
+                        <citedRange>16.571</citedRange>
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:270]: 16.0571

File: inscriptions/ISic030013.xml [hunk 34]
```diff
@@ -218,10 +260,53 @@
...
+                        <author>Lopez-Jimeno</author>
+                        <date>1991</date>
+                        <citedRange>no.19</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/N5IF2DZR"/>
+                    </bibl>
+                  
                 </listBibl>
-                <listBibl type="discussion"><bibl></bibl>
+                <listBibl type="discussion">
+                    <bibl>Jordan 104</bibl>
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:294]: ? delete?

File: inscriptions/ISic030013.xml [hunk 35]
```diff
@@ -218,10 +260,53 @@
...
+                    <bibl>Curbera 27</bibl>
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:295]: detele?

File: inscriptions/ISic030013.xml (hunk 28 above)
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:189]: add around all the names: <persName><name>Νίκυλλος</name></persName> (change/check previous files)

File: inscriptions/ISic030009.xml [hunk 36]
```diff
@@ -251,7 +251,7 @@
                         <citedRange>no.9</citedRange>
                         <ptr target="http://zotero.org/groups/382445/items/N5IF2DZR"/>
                     </bibl>
-                    <bibl type="bulletin" n="SEG">x
+                    <bibl type="bulletin" n="SEG">
```
Comment (valentinamignosa) [inline inscriptions/ISic030009.xml:254]: 36.0085

**Review by valentinamignosa — CHANGES_REQUESTED** [review]
I have indicated some small edits in the files. I am available (here or by email) if anything is not clear. Excellent work!

File: inscriptions/ISic030010.xml (hunk 6 above)
Comment (valentinamignosa) [inline inscriptions/ISic030010.xml:196]: delete final full-stop (generally stripping out modern punctuation)

File: inscriptions/ISic030013.xml (hunk 30 above)
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:191]: delete commas (generally stripping out modern punctuation)

File: inscriptions/ISic030013.xml [hunk 37]
```diff
@@ -177,38 +180,77 @@
...
                 <ab>
-                    <lb n="1"/>
-                    <lb n="2"/>
+                    <lb n="1"/> ἐ<unclear>π</unclear>ὶ ἐξο̄<unclear>λ</unclear>είαι καὶ αὐτο͂ν καὶ γενεᾶς· Νίκυλλος
+                    <lb n="2"/>Καπόσο̄, Δένδιλος Μάμονος, Αἴνο̄ν Β<unclear>λ</unclear>έπονος,
+                    <lb n="3"/>Ξένιος Ἀπόντιος ℎɛ̄ρακλείδας, Σαῦρις, <seg style="textdirection:r-to-l">ανις
+                   <lb n="4" break="no"/>Ἄθ</seg> Ταμμάρο̄ ℎɛ̄ρακλείδας, Ῥας Φαρμαυα, <seg style="textdirection:r-to-l">ν
+                    <lb n="5" break="no"/>Δίο̄</seg> Πιάκιος, Πιθθίας, Χαίο̄ν, <choice><orig>Μαμαρειṑι</orig><reg>Μαμμαρειṑι</reg></choice> <seg style="textdirection:r-to-l">τα
+                    <lb n="6" break="no"/>Ζο<unclear>ζ</unclear></seg>, Ἀγάθυλλος Ξένιος ℎɛ̄ρακλείδας, <seg style="textdirection:r-to-l">τος
+                    <lb n="7" break="no"/>Σύνε</seg> Ξένο̄νος.
```
Comment (valentinamignosa) [inline inscriptions/ISic030013.xml:195]: delete final full-stop

File: inscriptions/ISic030010.xml (hunk 8 above)
Comment (savannahneibart) [reply inscriptions/ISic030010.xml:220]: Sorry, what should be used here? I don't think I see what you mean.

Comment (valentinamignosa) [reply inscriptions/ISic030010.xml:207]: you can use one `<p>` element for the entire commentary. you can open `<p>` at the beginning of the commentary and close it at the very end, without entering different `<p></p>` for different lines

**Comment by JonPrag** [comment]
Hi Savannah, very good to chat earlier. I've gone through these and they look very good. I've made a number of minor tweaks and edits here and there - a few things to look out for in future:

- repository section: the 'settlement' here should be the town/city in which the museum or equivalent is situated, nothing to do with the provenance of the inscription.
- bibliography - when pulling a bibl item from Zotero, do make sure that it's the right one - there are commonly multiple articles by same author in same year; and not everything is necessarily in there yet.
- greek 'heta', make sure to use the unicode character Ͱ (https://unicode-table.com/en/0370/) or ͱ (https://unicode-table.com/en/0371/) not latin 'h' or similar.
- I tweaked the respStmt section, as Thea shoudl still be getting credit for gathering all the data originally in ISicDef.
- persName tagging: when a patronym (or papponym, etc.) appears as part of an individual's name, then that name should itself be tagged with persName type="attested" also, since this is the evidence for another individual, as well as being a patronymic, or similar (so persName inside another persName).

Many thanks again - and will follow up with some more options, but do feel free to continue with the curse tablets for now!

**Comment by savannahneibart** [comment]
Thank you! I will keep all of these notes in mind as I continue to work
through the curse tablets.

**Pull request merged**